frontend/src/app/commonComponents/TestResultGuidance/SyphilisResultGuidance.test.tsx

```diff
@@ -0,0 +1,40 @@
+import { render, screen } from "@testing-library/react";
+import { cloneDeep } from "lodash";
+
+import "../../../i18n";
+
+import { MULTIPLEX_DISEASES, TEST_RESULTS } from "../../testResults/constants";
+
+import SyphilisResultGuidance from "./SyphilisResultGuidance";
+
+describe("SyphilisResultGuidance", () => {
+  const mockResultTemplate = {
+    disease: {
+      name: MULTIPLEX_DISEASES.SYPHILIS,
+    },
+    testResult: TEST_RESULTS.POSITIVE,
+  } as MultiplexResult;
+
+  it("displays guidance for a positive RSV result", () => {
+    const { container } = render(
+      <SyphilisResultGuidance result={mockResultTemplate} />
+    );
+    expect(container).toMatchSnapshot();
+  });
+
+  const nonPositiveCases = [
+    TEST_RESULTS.NEGATIVE,
+    TEST_RESULTS.UNDETERMINED,
+    TEST_RESULTS.UNKNOWN,
+  ];
+
+  test.each(nonPositiveCases)(
+    "displays no RSV guidance for %p result",
+    (testResult) => {
+      const mockResult = cloneDeep(mockResultTemplate);
+      mockResult.testResult = testResult;
+      render(<SyphilisResultGuidance result={mockResult} />);
+      expect(screen.queryByText("For Syphilis")).not.toBeInTheDocument();
```

Comment:
I think this test will always be true because we changed "Syphilis" to lowercase here: https://github.com/CDCgov/prime-simplereport/pull/7946/files#diff-85309ebe46aefc6752792f3c5816128d0020bcc731a09f41984deee7c6c8f2beR415

Reply:
you were right! I also needed to include the colon